numpy/distutils/tests/test_fcompiler_nagfor.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import division, absolute_import, print_function
+
+from numpy.testing import assert_, run_module_suite
+
+import numpy.distutils.fcompiler
+
+nagfor_version_strings = [('nagfor', 'NAG Fortran Compiler Release '
+                           '6.2(Chiyoda) Build 6200', '6.2'),
+                          ('nagfor', 'NAG Fortran Compiler Release '
+                           '6.1(Tozai) Build 6136', '6.1'),
+                          ('nagfor', 'NAG Fortran Compiler Release '
+                           '6.0(Hibiya) Build 1021', '6.0'),
+                          ('nagfor', 'NAG Fortran Compiler Release '
+                           '5.3.2(971)', '5.3.2')]
+
+nag_version_strings = [('nag', 'NAGWare Fortran 95 compiler Release 5.1'
```

Comment:
That's a heck of a version string. Should there be a ',' at the end of the first line?

Reply:
Does `nag_version_strings` get used anywhere?

Reply:
Yep the version string is ridiculous but it's correct. The old compiler outputted it's own release as well as all the ones before it.
Fixed the test now to include all the version strings now, good spot.